app/events/[slug]/page.tsx

```diff
@@ -1,16 +1,181 @@
 import { getEvents, getEventBySlug } from "@/app/utils/contentful";
+import Image from "next/image";
+import dynamic from "next/dynamic";
+import { Document } from "@contentful/rich-text-types";
+import renderRichTextToReactComponent, {
+  ClassNames,
+} from "@/app/utils/rich-text";
+import {
+  EventCardType,
+  NestedAssetType,
+  MinimalCardType,
+} from "@/app/constants/types";
+import MinimalCard from "@/app/components/MinimalCard";
+import { adaptEventToMinimalCardType } from "../page";
 
 export async function generateStaticParams() {
   const events = await getEvents();
   return events.map((evt) => ({ slug: evt.slug }));
 }
 
+// DATE TIME FORMATTING
+function formatDateTime(date: Date): string {
+  const options: Intl.DateTimeFormatOptions = {
+    month: "long",
+    day: "numeric",
+    year: "numeric",
+    hour: "numeric",
+    minute: "numeric",
+    hour12: true,
+  };
+  return new Intl.DateTimeFormat("en-US", options).format(date);
+}
+
+const patientMinimalCardStyleProps = {
+  image:
+    "rounded-xl h-28 w-28 p-1.5 md:rounded-sm object-cover md:h-5/6 md:w-11/12 md:mx-auto md:px-4 md:pt-6",
+  wrapperDiv:
+    "md:rounded-lg md:w-1/5 md:border md:bg-white flex md:flex-col flex-reverse flex mb-3 md:h-full h-28 w-full rounded-lg bg-[#0076AD] md:mb-8",
+  header:
+    " md:text-center md:text-black md:pt-4 md:pb-6 md:text-2xl my-auto ml-6 text-3xl font-semibold tracking-wider text-white",
+  summary: "",
+};
+
+const eventMinimalCardStyleProps = {
+  image: "object-cover w-72 h-56 mt-3 rounded-xl mx-auto",
+  wrapperDiv: "mb-5 flex flex-col items-center h-3/5",
+  header: "w-64 font-bold text-lg py-4",
+  summary: "px-2 mb-4 w-72 overflow-hidden line-clamp-3 leading-tight",
+};
+
+const eventSummaryClassNames: ClassNames = {
+  paragraph: "text-base mb-4 mr-8 ml-1",
+};
```

Comment:
I think these style can go to the Minimal Card Component since it will be consistent throughout the site